docs/include/conf/portland/visiting.rst

```diff
@@ -52,6 +52,8 @@ Portland has a `separate streetcar network <http://www.portlandstreetcar.org/>`_
 
 If you are planning on using MAX, Portland Streetcar and/or city buses frequently during your stay, it may be worth investing in the `Hop Fastpass <https://myhopcard.com/>`__ fare card system.
 
+.. Note:: We recommend you use the `TriMet Trip Planner <https://trimet.org/#/planner>`__.
```

Comment:
Yeah, it is possible that the CSS style does not cater for the `Note`. Would you mind just incorporating the text into the paragraph instead of leaving as a Note?